scripts/import/import_variant_synonyms

```diff
@@ -103,16 +103,18 @@ elsif($source_name eq 'dbSNP') {
       '-driver'  =>  'mysql' 
     ); 
 
-  my $synonyms_output = query_synonyms_data($prod_dbc, 'rs_name,synonym_name,source_id', 'human_synonym'); 
+  my $sql_query = "SELECT rs_name,synonym_name,source_id FROM human_synonym"; 
+  my $synonyms_output = query_synonyms_data($prod_dbc, $sql_query); 
 
   my %data; # synonym_name => {rs_name} & {source_id} 
 
   # get data from human_synonym table (in production db) 
+  # some synonyms name are duplicated - from different sources 
   foreach my $result (@{$synonyms_output}){  
-    $data{$result->[1]}{rs_name} = $result->[0]; 
-    $data{$result->[1]}{source_id} = $result->[2]; 
+    $data{$result->[0].' '.$result->[1]}{rs_name} = $result->[0]; 
```

Comment:
Is this a unique set - do no sources use the same name for the same variant?

Reply:
$data{$result->[1]} is not unique because the same synonym name has different rsid - it was being overwritten.  
$data{$result->[0].' '.$result->[1]} is unique.

Reply:
This would be dangerous if we were adding further synonyms of this type to the production db, but as it is a static list it should not be a problem.